src/test/cypress/cypress/integration/1-RealTimeUsers.spec.js

```diff
@@ -70,6 +70,20 @@ describe ('RealTimeUsersPage',()=>{
         cy.get('.opfab-realtimeusers-entitiesgroups').eq(1).find('span').eq(0).should('have.text', 'EUROPEAN SUPERVISION CENTERS');
     })
 
+    it('Connection of operator1_fr, and check Real time users screen is ok, first name and last name should be displayed', ()=> {
+        opfab.loginWithUser('operator1_fr');
+        opfab.navigateToRealTimeUsers();
+
+        // we should have 9 disconnected entities and 1 connected (John Doe (operator1_fr) for ENTITY1_FR)
+        // we check the connected badge is for the second row
+        cy.get('.opfab-rounded-badge').should('have.length', 10);
+        cy.get('.opfab-blue-background').should('have.length', 1);
+        cy.get('.opfab-grey-background').should('have.length', 9);
+        cy.get('table').first().find('tr').eq(1).find('td').eq(0).find('.opfab-blue-background').should('have.length', 1);
+        cy.get('table').first().find('tr').eq(1).find('td').eq(0).find('span').eq(0).should('have.text', '1 ');
+        cy.get('table').first().find('tr').eq(1).find('td').eq(0).should('contain.text', 'John Doe');
+    })
+
```

Comment:
Replace this cypress test with a unit test in realtimeuser.view.spec.ts , that shall be suffisant 

for example add a last/first name to user4  and use it in a new test

Reply:
Done.